store/store_test.go

```diff
@@ -160,6 +191,9 @@ func TestStore_generateMongoQuery_allparams(t *testing.T) {
 			"$gte": "2015-10-07T15:00:00.000Z",
 			"$lte": "2015-10-11T15:00:00.000Z"},
 		"$and": []bson.M{
+			{"$or": []bson.M{
```

Comment:
These 3 lines should just be:
```suggestion
			{"uploadId": "xyz123"},
```

Reply:
Yep.  Will change this.